pkg/apis/scheduling/v1beta1/labels.go

```diff
@@ -46,3 +46,6 @@ const JDBMaxUnavailable = "volcano.sh/jdb-max-unavailable"
 
 // NumaPolicyKey is the key of pod nuam-topology policy
 const NumaPolicyKey = "volcano.sh/numa-topology-policy"
+
+// TopologyDecisionAnnotation is the key of topology decision about pod request resource
+const TopologyDecisionAnnotation = "volcano.sh/topology-decision"
```

Comment:
Where will this const be used? Suggest to give more introduction

Reply:
added  some introduction at the head